pagopa/io-app
# feat: [IOBP-311] Added newWallet paymentMethod selection screen

## Short description
added said screen, with access from the playgrounds section
<img src="https://github.com/pagopa/io-app/assets/60693085/4b24a3fa-311c-4781-b8a6-6eb92a378023)" width=200/>|<img src="https://github.com/pagopa/io-app/assets/60693085/7f6f0807-cbdb-496c-a431-ee8bdc899760)" width=200/>

## List of changes proposed in this pull request
- added said screen
- added required navigator entries
- added playground entry point
- mocked components and data where needed
- added required i18n keys

this screen is still in early-ish development as of writing this PR, as some components are missing/outdated from the DS

## How to test
navigate to the profile section, then to the new wallet playground and test the checkout flow to make sure the page behaves as expected
